readme.md

```diff
@@ -1,17 +1,17 @@
-![](learning_flow.png "Cartoon of this app's Mandarin learning flow. Nodes represent characters and edges connect characters which form words. The salmon section of the graph represents the expanding set of words and characters known by a learner.")
+![](learning_flow.png "Cartoon of this app's Mandarin learning flow. Nodes represent characters and edges represent words. The salmon section of the graph represents the expanding set of words and characters known by a learner.")
```

Comment:
ooo I like it! 

I missed the mouse over at first, is it possible to make it a caption? (have it appear right under the graph, like you see in scientific papers. (I am having trouble remembering what those are called) ie. Figure 1: ...)